typography-plugin.js

```diff
@@ -0,0 +1,73 @@
+const plugin = require("tailwindcss/plugin");
+const typography = plugin(function ({ addUtilities, theme }) {
+  const newUtilities = {
+    ".heading1": {
+      fontSize: theme("fontSize.mobileh1"),
+      fontWeight: theme("fontWeight.bold"),
+      fontFamily: theme("fontFamily.header"),
+      borderBottom: "4px solid #af3c43",
+      "@media (min-width: 500px)": {
```

Comment:
Thinking we should also be using our predefined breakpoints as well instead of hard coding